package.json

```diff
@@ -27,9 +27,9 @@
   "homepage": "https://github.com/auth0/auth0-deploy-cli#readme",
   "dependencies": {
     "ajv": "^6.5.2",
-    "auth0": "^2.17.0",
+    "auth0": "astanciu/node-auth0#hooksapi",
     "auth0-extension-tools": "^1.4.4",
-    "auth0-source-control-extension-tools": "~3.5.1",
+    "auth0-source-control-extension-tools": "^4.0.0-dev.6",
```

Comment:
Need to update, blocked by: https://github.com/auth0-extensions/auth0-source-control-extension-tools/pull/79